testsuite/features/secondary/srv_mainpage.feature

```diff
@@ -13,26 +13,13 @@ Feature: Main landing page options and preferences
     Then I should see a "Sign In" text
 
 @uyuni
-  Scenario: Access the About page
+  Scenario: Access the API Documentatione
     Given I am not authorized
     When I go to the home page
-    And I follow "About"
-    Then I should see a "About Uyuni" text
-
-  Scenario: Access the Copyright Notice
-    Given I am not authorized
-    When I go to the home page
-    And I follow "Copyright Notice"
-    Then I should see a "Copyright (c) 2011 - 2023 SUSE LLC." text
```

Comment:
We will not cover anymore that our copyright notice is correct, at least on that line?